tests/components/aquacell/test_config_flow.py

```diff
@@ -0,0 +1,111 @@
+"""Test the Aquacell config flow."""
+
+import string
+from unittest.mock import AsyncMock, patch
+
+from aioaquacell import ApiException, AuthenticationFailed
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.aquacell.const import DOMAIN
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.components.aquacell import TEST_RESULT_DATA, TEST_USER_INPUT
+
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
+
+
+async def __mock_authenticate(self, user_name, password) -> string:
+    return "refresh-token"
```

Comment:
Please create a test fixture for the `AquacellApi`, check `epion`, `analytics_insights` for examples